### PR TITLE
Fix logic in allFalse to return false on empty slice

### PR DESCRIPTION
**Pull request by WillSewell**
@mdpye look ok?

### CODE REVIEW

File: cmd/buddha.go [hunk 1]
```diff
@@ -351,6 +351,9 @@ func executeHealthCheck(wg *sync.WaitGroup, cmd buddha.Command, check buddha.Che
 }
 
 func allFalse(arr []bool) bool {
+	if arr == nil {
```
Comment (mdpye) [inline cmd/buddha.go]: Should we check nil or empty?

Comment (WillSewell) [inline cmd/buddha.go]: It's my understanding that the zero value of a slice (zero length) "acts" as `nil` (see: http://blog.golang.org/go-slices-usage-and-internals). I'm struggling to find precisely what this means in terms of whether it is internally the same thing, or whether they are equal in terms of equality. @jamescun?

Comment (zimbatm) [inline cmd/buddha.go]: `len(arr) == 0` would be better, it handles both the nil and empty cases

Comment (zimbatm) [inline cmd/buddha.go]: See https://play.golang.org/p/4DdNsgX9Jo

**Pull request merged**